lms/templates/goal_reminder_banner.html

```diff
@@ -1,11 +1,10 @@
 {# email client support for style sheets is pretty spotty, so we have to inline all of these styles #}
 {% if image_url %}
-<div class="goals-banner-div" style="
-    width: 100%;
-    height: 0;
-    padding-top: 42%;
-    background-image: url('{{image_url}}goalreminder-desktop-hero.png');
-    background-size: contain;
-    background-repeat: no-repeat;">
-</div>
+<table cellpadding="0" cellspacing="0">
```

Comment:
the above styling allowed me to sub in a smaller mobile image to reduce the size of the payload - however, i'm concerned after some testing that the above css might not work in some ancient email clients, so i switched to just using one image like we already do for our marketing emails